app/javascript/components/molecules/ContentSection.js

```diff
@@ -9,14 +9,14 @@ const ContentSection = ({ title, description, children }) => (
 )
 
 ContentSection.Header = ({ title, description }) => (
-  <div className='app-inner header-wide'>
+  <div className='app-inner'>
```

Comment:
removing all of these x-wide classes get rid of some pretty heavy left-padding to help the columns look OK. This impacts the paper app too but again I think it's OK